chains/solana/contracts/programs/ccip-offramp/src/context.rs

```diff
@@ -366,6 +378,66 @@ pub struct CommitReportContext<'info> {
     // [...chainConfig accounts] fee quoter accounts used to store gas prices
 }
 
+#[derive(Accounts)]
+#[instruction(_report_context_byte_words: [[u8; 32]; 2], raw_report: Vec<u8>)]
```

Comment:
Looks like these are not used to derive/constraint any PDA, could you remove this line please?